app/models/event_stream.rb

```diff
@@ -22,4 +22,6 @@ class EventStream < ApplicationRecord
   belongs_to :container_replicator
   belongs_to :container_group
   belongs_to :container_node
+
+  belongs_to :middleware_server, :foreign_key => :middleware_server_id
```

Comment:
To maintain the relation in a similar way other classes do.
This is the context:
https://github.com/lucasponce/manageiq/blob/8c277b7929ed598b4d5e0a959aa3ce29d6d1173e/app/models/event_stream.rb